pandas/core/internals/array_manager.py

```diff
@@ -923,15 +923,13 @@ def idelete(self, indexer) -> ArrayManager:
     # --------------------------------------------------------------------
     # Array-wise Operation
 
-    def grouped_reduce(self: T, func: Callable, ignore_failures: bool = False) -> T:
```

Comment:
i think you should be able to do the same in managers.py

Reply:
Indeed, thanks